doc/src/tutorial/matrices.rst

```diff
@@ -461,7 +461,7 @@ by injecting a custom zero test with warnings enabled.
     ...         result = None
     ...
     ...     # Warnings if evaluated into None
-    ...     if result == None:
+    ...     if not result:
```

Comment:
What will happen if `result` is `False`?

Reply:
Sorry, I took @kenluck2001's comment for granted, and applied changed without actually verifying the context.

Now that I looked into it, i see a problem with that code: it relies on exceptions for control flow and reuses `result` variable for two different purposes: flag indicating a mere presence of a value AND storage for a value itself. Submitting a patch in a minutes, hold on...